src/components/Header.js

```diff
@@ -4,31 +4,37 @@ import { NavLink } from "react-router-dom"
 import Logo from "../assets/logos/FlatFinderLogo.png"
 
 const Header = () => {
-	return (
-		<Nav className="me-auto">
-			<div className="header-navlink-left">
-				<NavLink className="navlink" to="/index">
-					Rent
-				</NavLink>
-				<NavLink className="navlink">List</NavLink>
-				<NavLink className="navlink">Manage</NavLink>
-			</div>
-			<div className="logo-container">
-				<NavLink to="/">
-					<img
-						className="header-logo-center"
-						src={Logo}
-						alt="flat finders logo"
-					/>
-				</NavLink>
-			</div>
-			<div className="header-navlink-right">
-				<NavLink className="navlink">Sign Up</NavLink>
-				<NavLink className="navlink sign-in">Sign In</NavLink>
-				<NavLink className="navlink">Logout</NavLink>
-			</div>
-		</Nav>
-	)
+  return (
+    <Nav className="me-auto">
+      <div className="header-navlink-left">
+        <NavLink className="navlink" to="/index">
+          Rent
+        </NavLink>
+        <NavLink className="navlink">List</NavLink>
+        <NavLink className="navlink">Manage</NavLink>
+      </div>
+      <div className="logo-container">
+        <NavLink to="/">
+          <img
+            className="header-logo-center"
+            src={Logo}
+            alt="flat finders logo"
+          />
+        </NavLink>
+      </div>
+      <div className="header-navlink-right">
+        <NavLink className="navlink" to="/signup">
+          Sign Up
+        </NavLink>
+        <NavLink className="navlink" to="/signin">
+          Sign In
+        </NavLink>
+        <NavLink className="navlink" to="/">
+          Logout
+        </NavLink>
```

Comment:
It looks like this is a nav link where it should be a button that triggers the signout fetch call.